styles/styles.css

```diff
@@ -0,0 +1,168 @@
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'Poppins', sans-serif;
+  font-size: 20px;
+  line-height: 30px;
+  color: #ffffff;
+  background-color: #1c1a1a;
+  padding: 0;
+  margin: 0 auto;
+  width: 1920px;
```

Comment:
```suggestion
  min-width: 1920px;
```
Тогда у тебя картинка львицы будет всегда по центру
![image](https://user-images.githubusercontent.com/79633923/190273884-331e8248-e146-4d86-9710-7eac10e9c44d.png)